### PR TITLE
Mailed messages

### PR DESCRIPTION
**Pull request by DutcherNL**
Implemented full urls
Small mail layout improvements

### CODE REVIEW

File: Dining/views.py [hunk 1]
```diff
@@ -306,15 +306,17 @@ def post(self, request, *args,**kwargs):
...
-            send_templated_mail(subject=subject, template_name=template, context_data=context, recipient=entry.user)
-
             success_msg = "The user is removed from the dining list"
 
         # Process deletion
         form = DiningEntryDeleteForm(entry, request.user, {})
         if form.is_valid():
             form.execute()
+
+            if entry != request.user:
```
Comment (mhvis) [inline Dining/views.py]: I think this is wrong

File: Dining/views.py [hunk 2]
```diff
@@ -306,15 +306,17 @@ def post(self, request, *args,**kwargs):
...
+                # Send mail to the people on the dining list
```
Comment (mhvis) [inline Dining/views.py]: It's always only 1 person right? (Namely the entry that is deleted)

File: Dining/views.py [hunk 3]
```diff
@@ -587,7 +593,7 @@ def delete(self, request, *args, **kwargs):
             messages.success(request, _("Dining list is deleted"))
 
             # Need to use reverse from the DiningListMixin superclass
```
Comment (mhvis) [inline Dining/views.py]: Remove this comment

File: Dining/views.py (hunk 1 above)
Comment (DutcherNL) [reply Dining/views.py]: Yup. Oops. That was too quick a fix haha

**Pull request merged**